src/main/java/frc/robot/subsystems/intake/IntakeIO.kt

```diff
@@ -0,0 +1,44 @@
+package frc.robot.subsystems.intake
+
+import edu.wpi.first.units.Measure
+import edu.wpi.first.units.Voltage
+import org.littletonrobotics.junction.LogTable
+import org.littletonrobotics.junction.inputs.LoggableInputs
+
+interface IntakeIO {
+    class IntakeInputs : LoggableInputs {
+        var intakeLinearVelocity: Double = 0.0
+        var intakeSupplyVoltage: Double = 0.0
+        var intakeMotorVoltage: Double = 0.0
+        var intakeStatorCurrent: Double = 0.0
+        var exitSensor: Boolean = false
+        var intakeSensor: Boolean = false
```

Comment:
These names are not that good. Is it just for one 20ms tick these are true? Their purpose should be made clear as booleans; currently they seem like they should be `DigitalInput`.